lms/envs/common.py

```diff
@@ -2268,6 +2268,7 @@
 STATIC_TEMPLATE_VIEW_DEFAULT_FILE_EXTENSION = 'html'
 
 SUPPORT_SITE_LINK = ''
+ACTIVATION_EMAIL_SUPPORT_LINK = 'https://support.edx.org/hc/en-us/articles/227340127-Why-haven-t-I-received-my-activation-email-'
```

Comment:
@douglashall No worries at all.

I think we should just default it to `''` and let it default to `SUPPORT_SITE_LINK` if the user doesn't set it. The edx-internal repo members will presumably handle that part of the equation on their own.